roxie/ccd/ccdmain.cpp

```diff
@@ -1004,6 +1005,7 @@ int STARTQUERY_API start_query(int argc, const char *argv[])
         tmpFlag |= _CRTDBG_CHECK_ALWAYS_DF;
         _CrtSetDbgFlag( tmpFlag );
 #endif
+        EnableSEHtoExceptionMapping();
```

Comment:
I do not see the need for this, it is already called elsewhere and without it we still get a core file from a SEGV.